compare_platforms/include/review.h

```diff
@@ -0,0 +1,151 @@
+/**
+ * Review struct and auxiliary functions to display review's details,
+ * change the content, and operate on the struct's permissions.
+ *
+ * This is based on the employee example at
+ * https://github.com/capablevms/cheri-examples/blob/master/employee/include/employee.h
+**/
+
+#ifdef __CHERI_PURE_CAPABILITY__
+#include "../../include/common.h"
+#endif
+
+#include <assert.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+
+/**
+ * Reviewer's username, real name, private comments, public review, and permissions. 
+**/
+struct review
+{
+    char *username;
+    char *realname;
+    char *privatereview;
+    char *publicreview;
+    char *permissions;
+};
+
+/**
+ * Check this review's permissions.
+ * param: review, the struct with the details of the review whose permissions we are checking
+ * return: true if the review has write permissions, false otherwise.
+**/
+bool can_write(struct review *rv) {
```

Comment:
I think this function can be just `return strchr(rv->permissions, "w") == NULL;` ?

Reply:
On lines 46-48, can_write has been reduced to:

return (strchr(rv->permissions, 'w') != NULL);

Reply:
> 

Fixed in a8a7797 (a8a7797167ed886112d4223145091bc1b7a54949)

Reply:
> On lines 46-48, can_write has been reduced to:
> 
> return (strchr(rv->permissions, 'w') != NULL);

Fixed in 84d32d463865d0ff38a349e845ffbed0ea7cd48a

Reply:
> I think this function can be just `return strchr(rv->permissions, "w") == NULL;` ?

Fixed in a41b3e55231a43e96a0da7612f43668dd452f796